meta/main.yml

```diff
@@ -4,11 +4,11 @@ galaxy_info:
   description: Formats and mounts a storage partition
   company: Open Microscopy Environment
   license: BSD
-  min_ansible_version: 2.1
+  min_ansible_version: 2.10
```

Comment:
Maybe then leave this at 2.1 ? As discussed this morning @jburel - I cannot see any dependencies here.